prprocessor/__main__.py

```diff
@@ -395,6 +396,8 @@ async def on_pr_modified(*, action: str, pull_request: Mapping, **_kw) -> None:
         labels.add(Label.DEB)
     elif target_branch.startswith('rpm/'):
         labels.add(Label.RPM)
+    if STABLE_PACKAGING_BRANCH_REGEX.match(target_branch):
```

Comment:
could also be added in L392, wasn't sure what's more readable.

Reply:
I was thinking about moving it to the config. We already have it elsewhere. Any thoughts on that?

Reply:
moving what exactly?

Reply:
Initially I thought about `stable_branch_prefix` and `stable_branch_suffix` in `config.yaml` but ended up with https://github.com/theforeman/prprocessor/pull/187 which should also fix "Fixed in Releases" for packaging PRs.